packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1548,6 +1565,10 @@ export const components: ThemeOptions["components"] = {
     },
   },
   MuiTableContainer: {
+    defaultProps: {
+      // @ts-expect-error valid prop and value; MUI TS bug
+      component: "figure",
+    },
```

Comment:
```suggestion
    defaultProps: {
      // @ts-ignore valid prop and value; MUI TS bug
      component: "figure",
    },
```

I would use ts-ignore here because we're not expecting an error, we're ignoring it. Expecting an error is where you know better than TS. They're the same thing effectively, but the semantic meaning is important.